wagtailio/project_styleguide/templates/patterns/components/search.html

```diff
@@ -0,0 +1,47 @@
+{% extends "patterns/base_page.html" %}
```

Comment:
suggestion: what do you think to naming this file `search-results.html`?

Reply:
Sounds great to me! Totally forgot you had been doing the search component in tandem